src/cljam/util/sam_util.clj

```diff
@@ -330,8 +334,8 @@
 (defn ref-name
   "Returns a reference name from the reference ID. Returns nil if id is not
   mapped."
-  [refs id]
-  (if (<= 0 id (dec (count refs)))
+  [refs ^long id]
+  (when (<= 0 id (dec (count refs)))
```

Comment:
I prefer `when` because one can tell at a glance that the function may return `nil`.

In [clojure.core.clj](https://github.com/clojure/clojure/blob/master/src/clj/clojure/core.clj), `if`s without third arg are less than 1% of entire use of `if`, while `when` with  single body is commonly used.